locksmith/__tests__/operations/transactionOperations.test.js

```diff
@@ -0,0 +1,59 @@
+import {
+  findOrCreateTransaction,
+  getTransactionsBySender,
+} from '../../src/operations/transactionOperations'
+
+const Sequelize = require('sequelize')
+
+const models = require('../../src/models')
+
+let Transaction = models.Transaction
+const Op = Sequelize.Op
+
+beforeEach(() => {
+  Transaction = models.Transaction // resetting Lock for before each test
+})
+
+describe('lockOperations', () => {
+  describe('findOrCreateTransaction', () => {
+    it('should call findOrCreate on the transaction object', async () => {
+      expect.assertions(5)
+      const transactionHash =
+        '0x7d6289db59c3434b6a034b4f211be52ca34f05e2aa856fc3e69b8af101355842'
+      const sender = '0x0x77cc4f1fe4555f9b9e0d1e918cac211915b079e5'
+      const recipient = '0xca750f9232c1c38e34d27e77534e1631526ec99e'
+      Transaction.findOrCreate = jest.fn(query => {
```

Comment:
I'm a bit concerned about us mocking out these methods that we don't own. I kind of glazed over this in LockOperation but do think we might need to consider some other options. 

The test know deeply about the implementation of these methods, if we change the implementation we'll have to update the tests as well

Reply:
This is right. I think eventually we should have one more level between the 'orm-models' and the actual app models.